internal/pkg/warewulfd/api/power.go

```diff
@@ -0,0 +1,200 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/swaggest/usecase"
+	"github.com/swaggest/usecase/status"
+	"github.com/warewulf/warewulf/internal/pkg/bmc"
+	"github.com/warewulf/warewulf/internal/pkg/node"
+	"github.com/warewulf/warewulf/internal/pkg/wwlog"
+)
+
+// PowerResult is the result structure for single node power status operations. get/set power status
+type PowerResult struct {
+	NodeId string `yaml:"node_id" json:"node_id" description:"ID of the node for which power status is being reported"`
+	BmcIp  string `yaml:"bmc_ip" json:"bmc_ip" description:"BMC IP address of the node"`
```

Comment:
`node_id` and `bmc_ip` aren't used in the cockpit PR. They should be removed here.

https://github.com/ctrliq/warewulf-cockpit/pull/16